React/CoreModules/RCTAppearance.mm

```diff
@@ -89,7 +89,9 @@ - (dispatch_queue_t)methodQueue
 
 RCT_EXPORT_SYNCHRONOUS_TYPED_METHOD(NSString *, getColorScheme)
 {
-  _currentColorScheme = RCTColorSchemePreference(nil);
+  if (_currentColorScheme == nil) {
+    _currentColorScheme = RCTColorSchemePreference(nil);
+  }
```

Comment:
While this change probably makes sense, it doesn't solve the issues with `overrideUserInterfaceStyle` in a brownfield app for me.

Reply:
Hey @koenpunt, if you have time to throw up a repo with a brownfield project that's broken (or hit me with some repro steps) I could probably take a look at brownfield a bit deeper sometime this week 👍 .